pkg/didcomm/protocol/introduce/metadata.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package introduce
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/common/service"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/didexchange"
+)
+
+const (
+	metaContextID    = "context_id"
```

Comment:
ContextID is a fairly generic term (and the term is already used in other situations). Perhaps rename or at least comment on its usage (in the code).

Reply:
I believe this is the identifier of a particular instance (or execution) of the state machine?

Reply:
Yes, it is. Context ID is an identifier witch helps protocol to determine that few participants are `on the same page`.

Reply:
We also use "Context" as the name of dependencies injected into protocols and golang has a context package.

It would be nice to use some other name.

Reply:
Changed it to `psmID` (protocol state machine identifier)